Xamarin.PropertyEditing.Mac/PropertyEditorPanel.cs

```diff
@@ -26,7 +27,7 @@ public PropertyEditorPanel (IHostResourceProvider hostResources)
 		}
 
 		// Called when created from unmanaged code
-		public PropertyEditorPanel (IntPtr handle) : base (handle)
+		public PropertyEditorPanel (NativeHandle handle) : base (handle)
```

Comment:
These `NativeHandle` changes mean we cannot merge this until we're ready to build VSMac with a newer macos workload.